### PR TITLE
fix(stream): Fix issue activity url [ISSUE-278]

### PR DESCRIPTION
**Pull request by billyvg**
Stream does not have groupId from router params, it comes from props

### CODE REVIEW

**Review by lynnagara — APPROVED** [review]
thanks

**Review by Chrissy — APPROVED** [review]
as long as `id` is already getting passed in here (which seems v likely) this lgtm;

**Comment by lynnagara** [comment]
Oh why the `id` change actually? i thought it was just the trailing slash that was missing

**Comment by billyvg** [comment]
Ah looks like this will break for `SimilarItem`.

**Comment by billyvg** [comment]
This does not work on the issues stream because previously it was looking for groupId through the router (when there is no group in context).

**Comment by lynnagara** [comment]
Should we pull it out of this.props.group?

File: src/sentry/static/sentry/app/components/stream/group.jsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ const StreamGroup = createReactClass({
         )}
         <GroupSummary w={[8 / 12, 8 / 12, 6 / 12]} ml={canSelect ? 1 : 2} mr={1} flex="1">
           <EventOrGroupHeader data={data} query={query} />
-          <EventOrGroupExtraDetails group {...data} />
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/stream/group.jsx]: `group` is not a prop on `<EventOrGroupExtraDetails>`

File: src/sentry/static/sentry/app/views/projectDashboard/eventNode.jsx [hunk 2]
```diff
@@ -25,10 +25,7 @@ const EventNode = createReactClass({
       <li className="group row">
         <div className="col-xs-8 event-details">
           <EventOrGroupHeader data={group} hideIcons hideLevel />
-          <EventOrGroupExtraDetails
-            lastSeen={group.lastSeen}
-            firstSeen={group.firstSeen}
-          />
+          <EventOrGroupExtraDetails {...group} />
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectDashboard/eventNode.jsx]: This also fixes:
image

**Review by lynnagara — APPROVED** [review]

File: src/sentry/static/sentry/app/views/projectDashboard/eventNode.jsx (hunk 2 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/projectDashboard/eventNode.jsx]: vs 
image

**Pull request merged**